### ISSUE TITLE
Add 2 Required Log Analytics Solutions for SQL Assessments for MDFC

### ISSUE THREAD

**Issue by jtracey93** [Azure/terraform-azurerm-caf-enterprise-scale#387]
> See upstream issue https://github.com/Azure/Enterprise-Scale/issues/979

Add 2 Required Log Analytics Solutions for SQL Assessments for MDFC.

**Missing Solutions:**
- SQLVulnerabilityAssessment 
- SQLAdvancedThreatProtection

```json
"product": "OMSGallery/SQLAdvancedThreatProtection",
"publisher": "Microsoft"

&

"product": "OMSGallery/SQLVulnerabilityAssessment",
"publisher": "Microsoft"
```

**Comment by krowlandson** [comment]
@jtracey93 Do you think these should be covered under the existing [enable_solution_for_sql_assessment](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/blob/246cf7de44369b2a40a1b07fafdd8b8b792e7c35/variables.tf#L90) input variable, or need dedicated ones?

https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/blob/246cf7de44369b2a40a1b07fafdd8b8b792e7c35/variables.tf#L75-L95

**Comment by jtracey93** [comment]
@krowlandson whilst I agree they could fit under the `enable_solution_for_sql_assessment` input variable I think they should be separate in an effort to provide as much felxibility as possible, just in case someone needs to not have one of the solutions but does want the other.

**TLDR: separate 👍**
